charts/superset/values.yaml

```diff
@@ -0,0 +1,57 @@
+image:
+  repository: apache/superset
+  pullPolicy: IfNotPresent
+  tag: 4.0.1
+
+storageSize: 5Gi
+
+secretName: superset-secrets
+secretKeySecretKey: secret
+adminPasswordSecretKey: admin-password
+
+entrypoint: |
```

Comment:
What was the thinking behind sticking this in the values file rather than in one of the templates?